MetaMask/metamask-mobile
# fix: harmony network image

## **Description**

Fixes the harmony network image.  It was showing the base network image.

## **Related issues**


## **Manual testing steps**

1. Add harmony network as a custom network
    - https://api.harmony.one
    - chain id: 1666600000
3. Verify correct network icon

## **Screenshots/Recordings**



### **Before**

![image](https://github.com/user-attachments/assets/17d1290e-8681-45b9-b3dc-6b40d05bbe79)

### **After**

<img width="365" alt="Screenshot 2024-08-13 at 8 21 04 AM" src="https://github.com/user-attachments/assets/71762e0e-8dd8-414d-8296-4e55d1002bbf">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
